app/javascript/containers/PlayerShowContainer.js

```diff
@@ -55,6 +164,9 @@ class PlayerShowContainer extends Component {
         apg={this.state.stat.apg}
         rpg={this.state.stat.rpg}
       />
+      {errorDiv}
+      {commentForm}
+
       <div className='comments'>
         {comments}
       </div>
```

Comment:
This file is huge.  I would highly recommend extracting everything having to do with the comments into a subcomponent except what's absolutely necessary for this component.